buildSrc/src/main/kotlin/otel.japicmp-conventions.gradle.kts

```diff
@@ -64,13 +66,29 @@ class AllowDefaultMethodRule : AbstractRecordingSeenMembers() {
   }
 
   /**
-   * Check if the change is related to LogEmitterProvider name change to LoggerProvider.
+   * Check if the change is related to log related renames.
    */
   // TODO(jack-berg): remove after 1.19.0.
-  fun isLogEmitterProvider(member: JApiCompatibility): Boolean {
+  fun isLogRename(member: JApiCompatibility): Boolean {
     try {
-      return member is JApiMethod &&
-        member.oldMethod.get().methodInfo.name.matches("(getSdk|set|add)LogEmitterProvider(.*)".toRegex())
+      if (member is JApiMethod) {
+        val longName = member.oldMethod.get().longName
+        return longName.matches(".*OpenTelemetrySdk\\.getSdkLogEmitterProvider.*".toRegex()) ||
+          longName.matches(".*OpenTelemetrySdkBuilder\\.setLogEmitterProvider.*".toRegex()) ||
+          longName.matches(".*AutoConfigurationCustomizer\\.addLogExporterCustomizer.*".toRegex()) ||
+          longName.matches(".*AutoConfigurationCustomizer\\.addLogEmitterProviderCustomizer.*".toRegex()) ||
+          longName.matches(".*ConfigurableLogExporterProvider.*".toRegex()) ||
+          longName.matches(".*SystemOutLogExporter.*".toRegex()) ||
+          longName.matches(".*OtlpJsonLoggingLogExporter.*".toRegex())
```

Comment:
i'm interested to understand why these are needed here even though they are in unstable artifacts?